tensorflow_addons/seq2seq/loss.py

```diff
@@ -15,18 +15,22 @@
 """Seq2seq loss operations for use in sequence models."""
 
 import tensorflow as tf
+from tensorflow_addons.utils.types import FloatTensorLike, TensorLike
+
+from typeguard import typechecked
+from typing import Callable, Optional
 
 
 def sequence_loss(
-    logits,
-    targets,
-    weights,
-    average_across_timesteps=True,
-    average_across_batch=True,
-    sum_over_timesteps=False,
-    sum_over_batch=False,
-    softmax_loss_function=None,
-    name=None,
+    logits: FloatTensorLike,
```

Comment:
```suggestion
    logits: TensorLike,
```
Yeah, maybe the name I chose (FloatTensorLike) is not very good. I means a single float scalar, not a tensor (or a tensor with a single scalar but I didn't find a way to enforce it). Maybe we should change the name to ScalarTensorLike?

Reply:
Thanks @gabrieldemarmiesse, I'll be sure of using it next time.